ibis/pandas/aggcontext.py

```diff
@@ -313,6 +314,92 @@ def compute_window_spec_interval(_, expr):
     return pd.tseries.frequencies.to_offset(value)
 
 
+def _window_agg_built_in(
+    frame, windowed, function, max_lookback, *args, **kwargs
+):
+    """Apply window aggregation with built-in aggregators.
+    """
+    assert isinstance(function, str)
+    method = operator.methodcaller(function, *args, **kwargs)
+
+    if max_lookback is not None:
```

Comment:
can this be an if/else

Reply:
I don't think needs an else statement. else is basically a noop (since method is already defined above)